### DISCUSSION TITLE
How to use curves to fix this picture?

### DISCUSSION THREAD

**Discussion by SebastianMpl** [bgrabitmap/lazpaint#588] [Q&A] [answered]
Hi,
Could you please send the screenshot of the curves tool with settings that fixes the following photo:

https://github.com/Apress/Beginning-Photo-Retouching-and-Restoration-Using-GIMP/blob/main/Chapter%204%20Practice%20Images/Ch4_ladies_on_patio.jpg

I don't know which curves tab to use - hue, saturation or lightness.
I've tried all but didn't get the results like in gimp with auto input colors.

Could you please advice?

Thank you

**Comment by circular17** [comment, marked as answer]
Hi @SebastianMpl,

Sure, here is how you can restore this photo. You'll notice that the photo is red, this correspond to a red channel having a minimum value different from zero. You can cancel that be flattening the curve at the beginning, up to the first bars you can see:

<img width="426" alt="flattening red" src="https://github.com/user-attachments/assets/382419a4-8fb5-4462-9da9-bc8d17472506">

There is a bit of the same with the blue channel:

<img width="427" alt="flattening blue" src="https://github.com/user-attachments/assets/e40b0188-c0b6-4b11-9d52-8f4e6dd3602f">

Regarding the green channel, it it seems the values are compressed to lower values. It is possible to compensate like this:

<img width="421" alt="raising green" src="https://github.com/user-attachments/assets/c9e8fc45-8201-4e58-bdf3-8b01c577a756">

Regarding lightness, it is a bit of both, there is some residual lightness (minimum value) and also it is a bit compressed:

<img width="423" alt="lightness" src="https://github.com/user-attachments/assets/20431bc5-622f-4187-85c0-5819a9409f93">

Saturation needs as well a little boost:

<img width="425" alt="saturation boost" src="https://github.com/user-attachments/assets/9efd0881-27b0-4e95-b957-eaaa9717fd5e">

After applying the curves, you can ensure it reaches the white color by applying "Colors > Normalize".

And maybe slight hue shift to clear the redness effect on the shadows of the skin:

<img width="486" alt="hue shift" src="https://github.com/user-attachments/assets/8b7e8f13-c407-4a63-9923-456dc439796e">

This will give the following result:

![Ch4_ladies_on_patio_balance](https://github.com/user-attachments/assets/e2f2430e-7e76-4ea9-a4de-10a895911c66)

I am curious what you get with Gimp.

Warm regards

**Reply by SebastianMpl** [reply]
Brilliant answer @circular17 !
Thank you very much.
I did all the steps from your screen and the result is great. Not yet so great as yours, but I'm closer.
It looks like I need to get more familiar with curves tool but repeating this  steps several times should do the trick.

With GIMP I used levels and get similar result to yours, but lazpaint is so small that now I can switch to it with all my needs.

Thank you once again.

**Reply by circular17** [reply]
I am glad you find the answer satisfying. Something that may make the process easier is to first do the RGB curves (red, green, blue), apply and then reopen the curve tool and do the HSL curves (hue, saturation, lightness). This way the histograms will reflect more accurately the actual distribution of values when you draw the curves.

You mention that LazPaint is so small and that is heartwarming for me, I like the idea.

Warm regards
